script/Arcueid/Day 5/05_00_ARC05_6_1.txt

```diff
@@ -36,17 +36,17 @@ And for that reason―――
 [sha:43a451103c99183a5f15e5c03da5f8db5471659b]{
 -- Page 452, Offset 9793.
 -- 「そうだ、怖じ気づいてる場合か……！」
-"That's right. This is no time to be scared...!"
+"That's right. This is no time to lose my nerve...!"
```

Comment:
I feel like ' This is no time to X' smells slightly JTE, maybe
> That's right. I can't lose my nerve/get cold feet now...!

Reply:
I don't really feel like its an unnatural expression but I don't mind changing